android/authentication.md

```diff
@@ -7,6 +7,13 @@ title: Authentication
 {% assign media_base = base_dir | append: page.dir | append: "media" %}
 {% assign common_media = base_dir | append: "/images" %}
 
+<br />
+
+**Note**
+This guide shows how to add Auth using the existing AWS Mobile SDK for Android and the Amplify CLI toolchain.
+Currently, this is our supported way of using Auth with Amplify Libraries for Android (preview).
```

Comment:
-> "Currently, this is our supported way of using Auth with Amplify Libraries for Android until we release the Auth category for Amplify for Android"